### PR TITLE
chore(curriculum): remove dsa blocks from JS modules

### LINKED ISSUE #633: Remove All DSA labs from main
As discussed in the meeting, we moved the DSA sections from JS into the python sections.

So as such, we need to update main and remove all of the following labs:

- Quick Sort lab
- roman numeral converter 
- telephone validator 
- cash register
- stacks labs
- linkedlist lab
- hashmap lab
- depth first search lab
- nth fibonacci number lab

### PR DESCRIPTION
**Pull request by Dario-DC**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes https://github.com/freeCodeCamp/CurriculumExpansion/issues/633

This PR removes labs, workshops and quizzes covering topics that will be taught in the Python section of the full-stack cert.

The `intro.json` has been updated to reflect the order in the sheet and reduce conflicts.

Back end related blocks have been removed here https://github.com/freeCodeCamp/freeCodeCamp/pull/57078

TODO: 
- [x] remove duplicated ids from `duplicatedProjectIds`

### CODE REVIEW

**Review by jdwilkin4 — APPROVED** [review]

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Actually, looking at the issues list, a few of these quiz PR's are still in review.
let's hold off of on removing those in this PR. 
Any hacktoberfest issue that is still open should not be removed 
https://github.com/freeCodeCamp/freeCodeCamp/issues/56170

At quick glance, that looks like the dynamic programming quiz and graphs and trees. I might be missing one, so I would double that open PR list to make sure.

Then in a separate PR once those hacktoberfest PR are merged in, we can remove them from the cert

**Comment by ilenia-magoni** [comment]
Hacktoberfest has ended, if there was stalling to allow for people to get their hacktoberfest badges, that time has ended

**Comment by jdwilkin4** [comment]
I feel like we should get this one approved and merged in.
It looks like they worked really hard on it. I would feel bad if we just closed it since they are a first timer.
https://github.com/freeCodeCamp/freeCodeCamp/pull/56442

Then it can be removed in this PR along with the others

**Review by jdwilkin4 — APPROVED** [review]

**Review by moT01 — APPROVED** [review]

**Pull request merged**
